docs/developers/v2/fork-yearn-ui.md

```diff
@@ -0,0 +1,229 @@
+# Fork Yearn UI
+
+This is a step-by-step guide on how to fork and customize [Macarena Finance](https://github.com/yearn/macarena-finance) which is a yearn UI made to be forked. Deploying your own UI makes you eligible to receive partner [profit-sharing](https://docs.yearn.finance/partners/introduction#profit-share-model) fees.
```

Comment:
Instead of a multiword link, you can do it this way:

This is a step-by-step guide on how to fork and customize [macarena.finance](https://macarena.finance)  which is a yearn UI made to be forked using the open-source [repository](https://github.com/yearn/macarena-finance). Deploying your own UI makes you eligible to receive partner [profit-sharing](https://docs.yearn.finance/partners/introduction#profit-share-model) fees.